backend/routes.go

```diff
@@ -78,21 +81,33 @@ func (s *server) handleAddUser() http.HandlerFunc {
 //GET: endpoint for all articles
 func (s *server) handleGetArticles() http.HandlerFunc {
 	return func(w http.ResponseWriter, req *http.Request) {
+
```

Comment:
Empty lines are unnecessary at the top. At most, we should have one empty line between every logically independent piece of code. Smaller functions are easier to look at :)